e2e-tests/IIoTPlatform-E2E-Tests/TestModels/PublisherExtensions.cs

```diff
@@ -23,12 +23,13 @@ public static class PublisherExtensions {
             }
             return new PublishNodesRequestApiModel {
 
+                DataSetWriterGroup = model.DataSetWriterGroup,
```

Comment:
We can also add assignment of `DataSetPublishingInterval` here.

Reply:
addes